rules/do-you-use-the-best-static-site-tech-stack/rule.md

```diff
@@ -41,35 +41,10 @@ There are a few hosting options to choose from including:
 *	[Surge.sh](https://surge.sh/)
 
 There are also many different static site generators:
-  
-*	[Gatsby](https://www.gatsbyjs.com/) (recommended)
 
-    *	React based
-  
-    *	Lots of plugins
-    
-    * Data handling via GraphQL
-    
-    * Works natively with GitHub as a Datasource.
-  
-*	[Jekyll](https://jekyllrb.com/)
-    
-    * Liquid based [(Shopify’s templating language)](https://www.shopify.com/partners/blog/115244038-an-overview-of-liquid-shopifys-templating-language)
-    
-    * Simple to setup
-    
-    * Out of the box integration with GitHub pages
-    
-    * Scales poorly
-    
-    * [Windows is not supported](https://jekyllrb.com/docs/installation/windows/)
-  
-*	[Next.js](https://nextjs.org/)
-    
-    * React based
-    
-    * Server-side rendered pages but can be exported as a static site
-    
-    * Provides full control over which data handling technology you use
-    
-    * More scalable than conventional static site generators
+| **Name** | [Gatsby](https://www.gatsbyjs.com/) | [Jekyll](https://jekyllrb.com/) | [Next.js](https://nextjs.org/) | [Scully](https://scully.io/) | [Gridsome](https://gridsome.org/) | [Statiq](https://statiq.dev/)
```

Comment:
The product order seems random - would be cool if it was ordered in some logical way

Reply:
Ordered it by JavaScript first and then .Net then Liquid

Gatsby is first because it is recommended.